roles/inject_openshift_kube_objects/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "Create the kube_objects directory"
+  shell: "mkdir -p {{ kube_objects_dir }}"
+
+- name: "Get the CVP private key"
+  copy:
+    src: "{{ ssh_private_key }}"
+    dest: "{{ kube_objects_dir }}/private_key"
+    mode: 0600
+  no_log: true  
+
+- name: "Decode the symmetric key received thru UMB message"
+  shell: "echo \"{{ isv_symm_key }}\" | openssl enc -d -base64 > {{ kube_objects_dir }}/dec_sym_key"
+  no_log: true
+  
+- name: "Decrypt the symmetric key using CVP private key"
+  shell: "openssl rsautl -decrypt -oaep -inkey {{ kube_objects_dir }}/private_key -in {{ kube_objects_dir }}/dec_sym_key -out {{ kube_objects_dir }}/secret.key"
+
+- name: "Read the encrypted kube_objects and store it in a file"
+  shell: "echo \"{{ kube_objects }}\" | tr \" \" \"\n\" > {{ kube_objects_dir }}/enc_kube_object.txt"
+  no_log: true
+  
+- name: "Decode the kube_objects received thru UMB message"
+  shell: "cat {{ kube_objects_dir }}/enc_kube_object.txt | openssl enc -d -base64 > {{ kube_objects_dir }}/dec_kubeObjects.txt"
```

Comment:
IMHO we can merge previous two into single one.
`echo '{{ kube_objects }}' | tr ' ' '\n' | base64 -d > > {{ kube_objects_dir }}/enc_kubeObjects.txt`

The utility base64 is part of coreutils so it is available everywhere :-) but you can use openssl as well

Reply:
`dec_kubeObjects.txt` might be a bit confusing because it is just base64 **dec**oded and not **dec**rypted